cluster/charts/universal-crossplane/values.yaml.tmpl

```diff
@@ -60,11 +59,11 @@ priorityClassName: ""
 
 resourcesCrossplane:
   limits:
-    cpu: 500m
-    memory: 1Gi
+    cpu: 100m
+    memory: 512Mi
   requests:
-    cpu: 250m
-    memory: 768Mi
+    cpu: 100m
+    memory: 256Mi
```

Comment:
These are the chart defaults and can be overridden for specific use cases during installation.
If we believe the new defaults are better, we should update for upstream.

Introduced with https://github.com/upbound/universal-crossplane/pull/262

/cc @avalanche123 @negz

Reply:
Happy to sync these now, but yeah it might be nice to bump upstream a little. In particular I think giving Crossplane at least half a CPU core seems plenty reasonable.